src/main/java/mate.academy/service/FileReaderService.java

```diff
@@ -1,7 +1,9 @@
 package mate.academy.service;
 
 import java.util.List;
+import mate.academy.lib.Component;
 
+@Component
```

Comment:
If I understand correctly, in the README.md file the task requires to inject  "Interface implementations" if they are marked with @Component annotation, but not the original interfaces. So, we should put @Component annotation in the respective implementation classes.

Reply:
> If I understand correctly, in the README.md file the task requires to inject "Interface implementations" if they are marked with @component annotation, but not the original interfaces. So, we should put @component annotation in the respective implementation classes.

that's right